components/rich_text_editor/mention_suggestion.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
this file/object would need to come from the product and it is how product can override any behaviour of the mention plugin. or like this it can expose the function that gets the query to return list of suggestions to be shown